subgraph/src/fleek-nfa.ts

```diff
@@ -423,23 +426,29 @@ export function handleTransfer(event: TransferEvent): void {
   }
```

Comment:
Do we need to save the owner entity above here?

Reply:
We only need to save entities when their metadata have been modified. In this handler, owner is saved after the changes.